olm/eclipse-che-preview-kubernetes/deploy/olm-catalog/csv-config.yaml

```diff
@@ -1,3 +1,7 @@
-role-paths: [ "generated/roles/role.yaml" ]
+role-paths:
+  - "generated/roles/role.yaml"
+  - "../../deploy/cluster_role.yaml"
+  - "../../deploy/cluster_role_che.yaml"
```

Comment:
Why do we need to add 3 separate cluster-role.yaml files ?
Why was `cluster_role.yaml` added here while it wasn't necessary previously for the `K8S` use-case ?

Reply:
Fixed, we use only one cluster role binding

Reply:
Thanks @AndrienkoAleksandr !